launch/data_matching_miro.launch

```diff
@@ -1,20 +1,20 @@
 <launch>
 	<!-- Global params -->
-	<rosparam param="data_load_dir">~/miro/data/office5</rosparam>
-	<rosparam param="data_save_dir">~/miro/data/office5_tests/28</rosparam>
+	<rosparam param="data_load_dir">~/miro/data/trial</rosparam>
+	<rosparam param="data_save_dir">~/miro/data/trial_repeat/test1</rosparam>
 	<rosparam param="wait_for_ready">true</rosparam>
 	<rosparam param="image_resize_width">115</rosparam>
 	<rosparam param="image_resize_height">44</rosparam>
 	<rosparam param="image_subsampling">1</rosparam>
-	<rosparam param="image_field_of_view_width_deg">175.2</rosparam>
+	<rosparam param="image_field_of_view_width_deg">140</rosparam>
 	<rosparam param="patch_size">(9,9)</rosparam>
-	<rosparam param="image_recognition_threshold">0.1</rosparam>
+	<rosparam param="image_recognition_threshold">0.01</rosparam>
```

Comment:
It's better not to make this parameter so low. If the image correlation is below this threshold, no correction is performed. Making this really low is likely to make the correction more noisy.

And like the others can you move these changes to a new file